### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by lawruble13**
- Navbar links open in new tab
- GitHub navbar links go to project page
- Sidebar CSS fix
- Renaming RTD links fix

### CODE REVIEW

**Comment by lawruble13** [comment]
Resolves #183.

**Review by saadrahim — APPROVED** [review]

**Comment by lawruble13** [comment]
Currently PRs link to the source fork in the navbar (e.g. to my fork for this PR). Do we want to change that behaviour @saadrahim?

**Comment by saadrahim** [comment]
> Currently PRs link to the source fork in the navbar (e.g. to my fork for this PR). Do we want to change that behaviour @saadrahim?

Can we do something better? Hard to say what to do.

**Comment by lawruble13** [comment]
Could link to the target repo instead? Most of the time that would be correct.

**Comment by saadrahim** [comment]
> Could link to the target repo instead? Most of the time that would be correct.

Sounds good.

File: src/rocm_docs/rocm_docs_theme/sections/header.html [hunk 1]
```diff
@@ -23,36 +23,19 @@
...
-                        <a class="nav-link top-level header-menu-links" href="mailto:dl.rocm-docs-core@amd.com" id="menu4" role="button" aria-expanded="false" >
-                            Feedback
+                    {% macro nav_button(name, url) -%}
+                    <li class="nav-item">
+                        <a class="nav-link top-level header-menu-links" href="{{ url }}" id="nav{{ name.lower()|replace(' ','-') }}" role="button" aria-expanded="false" target="_blank" >
+                            {{ name }}
                         </a>
                     </li>
+                    {%- endmacro -%}
+                    {{ nav_button("GitHub", theme_repository_url) }}
```
Comment (samjwu) [inline src/rocm_docs/rocm_docs_theme/sections/header.html:33]: how does Jinja know what `theme_repository_url` is? Is it defined somewhere?

**Review by samjwu — APPROVED** [review]

File: src/rocm_docs/rocm_docs_theme/sections/header.html (hunk 1 above)
Comment (lawruble13) [reply src/rocm_docs/rocm_docs_theme/sections/header.html:33]: Yes, here: https://github.com/RadeonOpenCompute/rocm-docs-core/blob/fb334b73e834a4f06959a687a7b2cbc8206c2868/src/rocm_docs/theme.py#L20
The theme option `repository_url` is available to the Jinja template as `theme_repository_url`.

**Comment by lawruble13** [comment]
Alright, I think this is okay to merge. @saadrahim have a quick look to validate I've got everything?

**Comment by saadrahim** [comment]
Looks Good to Me. The RTD links stuff is hard to check. I am ok with waiting to confirm that works.

**Pull request merged**